cli/stream_command.go

```diff
@@ -1719,28 +1800,42 @@ func (c *streamCmd) showStreamInfo(info *api.StreamInfo) {
 				state = append(state, fmt.Sprintf("%s operation behind", humanize.Comma(int64(r.Lag))))
 			}
 
-			fmt.Printf("              Replica: %s\n", strings.Join(state, ", "))
+			fmt.Printf("                   Replica: %s\n", strings.Join(state, ", "))
 
 		}
 		fmt.Println()
 	}
 
 	showSource := func(s *api.StreamSourceInfo) {
-		fmt.Printf("          Stream Name: %s\n", s.Name)
-		fmt.Printf("                  Lag: %s\n", humanize.Comma(int64(s.Lag)))
+		fmt.Printf("               Stream Name: %s\n", s.Name)
+
+		if s.SubjectTransformDest != "" {
+			if s.FilterSubject == "" {
+				fmt.Printf("            Subject Filter: %s\n", ">")
```

Comment:
`Subject` here is also redundant can just be `Filter` and `Transform`.  Actually I just dont think we need to show it here.

```
Replication:

                   Sources: ORDERS_1, Subject filter: js.in.orders_1, Subject transform: js.in.orders_X

Source Information:

               Stream Name: ORDERS_1
            Subject Filter: js.in.orders_1
         Subject Transform: js.in.orders_X
                       Lag: 0
                 Last Seen: 388ms
```

The first part is the configuration showing filters and such, the 2nd part is the state of the configured replication.

You can see in `nats s state` only the 2nd half is shown, we dont show the config in there so I would remove the `Subject Filter` and `Subject Transform` completely from `Source Information` (and perhaps rename `Source Information` to `Source State`)

Reply:
Fine with changing to just `Filter` and `Transform` however I think I would like to still have the filter and transform repeated in Source Information (now source State) this is because you can now source more than once from the same stream and in that case there will be two (or more) entries in Source Information with the same "Stream Name" and therefore it's nice to have the filter/transform info repeated there so you can tell which source information relates to which subject filter. So leaving the repeat in for the moment, let me know if you're ok with that or not.

Reply:
Yeah thats fair, so I wonder how we can communicate this without making this block bigger as a stream with many sources could already be pages full.

Maybe something like `Stream: ORDERS_1 Filter: x Transform: y` leaving lag and seen as is today? Kind of small improvement so might not be worth doing so I am ok with this either way, lets just drop the redundant `Subject`